app/components/alchemy/ingredients/picture_view.rb

```diff
@@ -46,10 +48,11 @@ def call
         output = caption ? img_tag + caption : img_tag
 
         if is_linked?
+          target = ingredient.link_target.presence
           output = link_to(output, url_for(ingredient.link), {
             title: ingredient.link_title.presence,
-            target: (ingredient.link_target == "blank") ? "_blank" : nil,
-            data: {link_target: ingredient.link_target.presence}
+            rel: link_rel_value(target),
+            target:
```

Comment:
When a link's target attribute is set to `blank` (without an underscore), clicking the link twice will open it twice in the same window. However, when the target is set to `_blank` (with an underscore), clicking twice will open the link in two separate new windows.
The `_blank` behavior is more predictable and is what users typically expect. Looking at line 51 of the code, it appears Alchemy is saving the value as "blank" (without the underscore), though I'm not certain if this is the intended behavior.